src/documents/templates/admin/documents/document/change_form.html

```diff
@@ -26,4 +30,37 @@
 		django.jQuery(".field-created input").first().attr("type", "date")
 	</script>
 
+	{# Hack to make preview appear next to form #}
```

Comment:
The following should work and be the intended way:

```jinja
{% block extrastyle %}
{{ block.super }}
<style> ... </style>
{% endblock %}
```

Make sure to add this outside of the `block footer` section. 🙂

Reply:
Thanks, I've changed the template accordingly.